service/src/main/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperService.java

```diff
@@ -167,10 +167,12 @@ private boolean responseExists(DirectionResponse response, Element<Direction> di
     public void addAssigneeDirectionKeyValuePairsToCaseData(String assignee,
                                                             List<Element<Direction>> directions,
                                                             CaseDetails caseDetails) {
-        if (assignee.equals("courtDirections")) {
-            caseDetails.getData().put(assignee.concat("Custom"), extractPartyResponse(assignee, directions));
-        } else {
-            caseDetails.getData().put(assignee, extractPartyResponse(assignee, directions));
+        if (!assignee.equals(ALL_PARTIES.getValue())) {
```

Comment:
Is there a reason why we are always passing around the String representation of the enum? Would it be simpler to just use the enum or would that change if it is better extend outside the scope of the ticket?